Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
```

Comment:
I have fixed this issue by removing one ring in "FwdHelp". The new geometry is available at: http://fcc-tklayout.web.cern.ch/fcc-tklayout/FCChh_Option3.v02/index.html